src/Backend/Core/Layout/Templates/base.html.twig

```diff
@@ -47,7 +47,7 @@
           </div>
         </div>
       </div>
-    </div>
+    </main>
     <div id="root-footer"></div>
```

Comment:
Maybe change this to a `<footer>` element?

Reply:
Footer is already used under that, for the real footer. This is just a placeholder for the sticky footer to work.